llvm/lib/Transforms/Utils/Local.cpp

```diff
@@ -3606,3 +3606,41 @@ bool llvm::inferAttributesFromOthers(Function &F) {
 
   return Changed;
 }
+
+DIExpression *llvm::getExpressionForConstant(DIBuilder &DIB, const Constant *C,
```

Comment:
We never check if `C` or `Ty` are null, we can make these references

Reply:
Passing them as references.